packages/react-native/React/Views/RCTModalHostViewManager.h

```diff
@@ -5,16 +5,9 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-#import <React/RCTConvert.h>
```

Comment:
We can make it non-breaking by leaving this import here, right?

Reply:
@cipolleschi yes, that works, I've put it back. 👍